README.md

```diff
@@ -33,122 +33,137 @@ export COUCH2PG_DOC_LIMIT=1000
 export COUCH2PG_RETRY_COUNT=5
 ```
 
-Run it locally with environment variables: `npm ci && node .`
+*NOTE:* Currently, cht-couch2pg only runs in node versions 8 , 10 and 12. Later versions of node have been known to fail.+
 
-### Running locally in interactive mode (no environment variables needed)
+To run the contents of this repository locally, first, export the above environment variables and then run:  `npm ci && node .`
 
-Run it locally in interactive mode: `npm ci && node . -i`
+### Running locally in interactive mode without environment variables
 
+Run it locally in interactive mode: `npm ci && node . -i`
 
 ## Running tests through docker-compose
 
 Run tests with:
 
-```
-docker-compose build --build-arg node_version=[node version] test
-docker-compose up
+```bash
+docker-compose build cht-couch2pg
+docker-compose run cht-couch2pg grunt test
 ```
 
-Then in another terminal:
+Run tests in interactive watch mode with: `docker-compose run test npm run watch`
```

Comment:
This command needs to be updated to call the correct service name and command. Currently returns:
```
docker-compose run test npm run watch
ERROR: No such service: test
```